scripts/verify-package-parity.sh

```
@@ -0,0 +1,121 @@
+
+if [[ $# -ne 2 ]]; then
+    echo "Usage: $0 <candidate-image-ref> <reference-image-ref>"
+    exit 2
+fi
+
+CANDIDATE_IMAGE="$1"
+REFERENCE_IMAGE="$2"
+
+WORKDIR_CREATED=0
+if [[ -n "${WORKDIR:-}" ]]; then
+    WORKDIR="${WORKDIR}"
```

Shellcheck (0.11.0)

[info] This variable is assigned to itself, so the assignment does nothing.

(SC2269)